packages/bson2/bson2.0.0.1/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+synopsis: "Bson format encoding/decoding for Ocaml"
+maintainer: "stephanos.tsoucas@gmail.com"
+authors: "Stephanos Tsoucas"
+license: "Apache-2.0"
+homepage: "https://github.com/nosman/Bson2"
+bug-reports: "https://github.com/nosman/Bson2/issues"
+depends: [
+  "ocaml" {>= "4.04"}
+  "ounit" {with-test}
+  "dune" {build}
```

Comment:
```suggestion
  "dune" {>= "1.11"}
```
Dune shouldn't be tagged as a build dependency (see #14266) and the `dune-project` file in your repository indicates your project requires at least dune 1.11